gcc-arm-none-eabi-49 is no longer available from homebrew; using gcc-arm-none-eabi

The build instructions specify prerequisites for Mac OS X, that include installing `gcc-arm-none-eabi-49` via a Homebrew tap. That version is no longer labelled as such, but it's possible to install `gcc-arm-none-eabi`.

When I do that, I get this version string:

```
Edwards-MacBook-Air:firmware emv$ arm-none-eabi-gcc --version
arm-none-eabi-gcc (GNU Tools for ARM Embedded Processors) 4.9.3 20150529 (release) [ARM/embedded-4_9-branch revision 227977]
```

which I think is correct.

---

Completeness:
- [x] Minimum test case added
- [x] Device, system and user firmware versions stated
- [x] Particle confirmed

@BracWebb I tried it as well but i am running into these issues:

`opt/local/arm-none-eabi/include/sys/time.h:38:19: error: field 'it_interval' has incomplete type
   struct  timeval it_interval;
                   ^
/opt/local/arm-none-eabi/include/sys/time.h:39:19: error: field 'it_value' has incomplete type
   struct  timeval it_value;
                   ^
lib/mbedtls/library/timing.c:66:20: error: field 'start' has incomplete type
     struct timeval start;
                    ^
lib/mbedtls/library/timing.c: In function 'mbedtls_timing_get_timer':
lib/mbedtls/library/timing.c:285:20: error: storage size of 'offset' isn't known
     struct timeval offset;
                    ^
lib/mbedtls/library/timing.c:285:20: error: unused variable 'offset' [-Werror=unused-variable]`

--- a/docs/dependencies.md
+++ b/docs/dependencies.md
@@ -22,7 +22,27 @@ message if the version is older than this.
 **OS X** users can install the toolchain with [Homebrew](http://brew.sh/):
 - `brew tap PX4/homebrew-px4`
 - `brew update`
-- `brew install gcc-arm-none-eabi-49`
+- copy/paste this in Terminal and press ENTER to create the proper Brew formula
+```
+echo -e "require 'formula'
+
+class GccArmNoneEabi49 < Formula
+  homepage 'https://launchpad.net/gcc-arm-embdded'
+  version '20150306'
+  url 'https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q1-update/+download/gcc-arm-none-eabi-4_9-2015q1-20150306-mac.tar.bz2'
+  sha256 '049469f06c35d1a7c6d333236bc62ed340531c4377b4e16ce8c5aa8d134e8e8b'
+
+  def install
+    ohai 'Copying binaries...'
+    system 'cp', '-rv', 'arm-none-eabi', 'bin', 'lib', 'share', \"#{prefix}/\"
+  end
+end" > /usr/local/Homebrew/Library/Taps/px4/homebrew-px4/gcc-arm-none-eabi-49.rb
+```
+
+- install it!
+```
+brew install gcc-arm-none-eabi-49
+```
 - `arm-none-eabi-gcc --version` (should now say v4.9.x)
 
 If you are upgrading an existing installation you will have to unlink and link your symblinks:

